src/app/app-client.ts

```diff
@@ -1175,4 +1176,125 @@ export class AppClient {
   ): Promise<CreateKeyFromExistingKeyAuthorizationsResponse> {
     return this.client.createKeyFromExistingKeyAuthorizations({ id });
   }
+
+  /**
+   * Retrieves user-defined metadata for an organization.
+   *
+   * @param id The ID of the organization
+   * @returns The metadata associated with the organization as a plain JS object
```

Comment:
```suggestion
   * @returns The metadata associated with the organization
```
The other docstrings don't mention this so I think we can remove it

Reply:
Thanks for calling this out!